app/build.gradle

```diff
@@ -62,6 +62,11 @@ dependencies {
     implementation "com.squareup.okhttp3:okhttp:$rootProject.okHttpVersion"
     implementation "com.squareup.okhttp3:logging-interceptor:$rootProject.loggingInterceptorVersion"
 
+    //GSON and GSON converter library to serialize & deserialize object automatically in network
+    //request
+    compile "com.google.code.gson:gson:$rootProject.gsonVersion"
```

Comment:
Isn't com.google.code.gson already available in the project? See https://github.com/elimu-ai/appstore/blob/91e58f588e1ceacf9a8b95e60626ea76b5ceea2b/app/src/main/java/ai/elimu/appstore/synchronization/AppSynchronizationActivity.java#L132